### DISCUSSION TITLE
An Ability to Move Score From Global Leaderboard into Local

### DISCUSSION THREAD

**Discussion by PolskiKapsel** [ppy/osu#25031] [Ideas] [answered]
Like the title says, I would like to share an idea to move scores from global leaderboard into a Local one. The imported score of course could be seen from in-game scoreboard. Main reason for this feature would be an ability to export Your old score that You don't have replay of into Local leaderboard. That way there is no need for players to remember their old score if they want to beat it. It would be also helpful if a certain score from top 50 looks reachable for the Player, but the idea of having 50 scores scrolling in-game seems too distracting.  
I'm uploading a screenshot edited with my bad GIMP skills to show how it would look like from song select menu. Of course the score from Global Leaderboard would have 1st place in this case.
![score_moved](https://github.com/ppy/osu/assets/102519385/f3ecb895-3b8f-4951-8fae-aecd32f9bad3)

**Comment by peppy** [comment, marked as answer]
You can download the replay from the global leaderboard. It will then show in your local scoreboard.

I guess you're wanting scores specifically without replays. I guess we could offer some kind of partial export for that.

**Reply by PolskiKapsel** [reply]
Pardon my Ignorance, however every time I tried to download a score it says the Replay is unavailable or the downloaded score becomes 0. I tried watching the replay in hope for the game to calculate it locally but with no success. Is this feature still in development or is it a bug from my side? 
 Here's a video showing what I meant: https://www.youtube.com/watch?v=5I0B_7ZPHDQ

**Reply by Theighlin** [reply]
Storing replays has only been a thing for roughly 4 months, so the vast majority of lazer scores can't be downloaded and played back.
For scores being 0 when downloaded, old replays do not have it set (it used to be calculated from hit statistics, can't be done anymore)

**Reply by PolskiKapsel** [reply]
Since some of the replays cannot be downloaded because is it relatively new thing on lazer, but the score itself is stored, then downloading just the score seems to be a reasonable feature... 
Even for the replays that cannot be calculated anymore, the Score itself is still stored on the osu! servers, since otherwise it wouldn't be visible on the leaderboards. Can't we just take that score from the server to have it displayed locally? Or is it too much of a niche feature I'm asking for?

**Reply by respektive** [reply]
On stable you were able to hit F2 even on scores without a replay available. It would download a .osr with just the plain score and no replay data, that you were able to import into your local scores then. Perhaps something like that could be reintroduced on lazer.

Should probably address https://github.com/ppy/osu/issues/20932 first though.
